python/hail/expr/column.py

```diff
@@ -0,0 +1,1192 @@
+from __future__ import print_function  # Python 2 and 3 print compatibility
+
+from hail.java import *
+from hail.htypes import *
+from hail.representation import *
+
+
+def to_expr(arg):
+    if isinstance(arg, Column):
+        return arg.expr
+    elif isinstance(arg, str):
+        return "\"" + arg + "\""
+    elif isinstance(arg, bool):
+        return "true" if arg else "false"
+    elif isinstance(arg, long):
+        return "{}.toInt64()".format(arg)
+    elif isinstance(arg, list):
+        return "[" + ", ".join([to_expr(a) for a in arg]) + "]"
+    elif isinstance(arg, set):
+        return "let xfsjd = [" + ", ".join([to_expr(a) for a in arg]) + "] in xfsjd.toSet()"
```

Comment:
Hmm, a bit dicy.  We should add a function on HailContext (which you can get through the Env) to keep a counter and generate unique symbols of the form `__gen_N` say.  This isn't foolproof but probably good enough for this prototype, and we can fix it properly when we're generating the AST directly.